### PR TITLE
complete translation lighthouse overview into korean

### PR DESCRIPTION
**Pull request by techhtml**
dear Fundanmentals.
Now i complete translation lighthouse overview into korean.
next i'll translate audits sections.

PTAL.
thanks.

### CODE REVIEW

File: src/content/ko/tools/lighthouse/index.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Learn how to set up Lighthouse to audit and improve your web apps.
+
+{# wf_updated_on: 2017-01-18 #}
```
Comment (petele) [inline src/content/ko/tools/lighthouse/index.md]: Please change this to the date you translated the doc

File: src/content/ko/tools/lighthouse/index.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+Note: Lighthouse는 현재 [홈스크린에 추가](/web/fundamentals/engage-and-retain/app-install-banners/)나 오프라인 지원 같은,
+프로그레시브 웹 앱 기능에 중점을 둡니다. 그러나 프로젝트의 가장 중요한 목표는 웹 앱 품질의 모든 측면에 대한 end-to-end 테스트를 제공하는 겁니다.
+
+## Lighthouse 설정하기
+
+Lighthouse는 크롬 확장프로그램, 커맨드 라인 도구로 실행할 수 있습니다.
+크롬 확장프로그램은 리포트를 읽는데 더 유저에 친숙한 인터페이스를 제공합니다.
+커맨드 라인 도구는 Lighthouse를 지속적 통합 시스템에 넣을 수 있습니다.
+
+### 크롬 확장프로그램 설치하기
```
Comment (petele) [inline src/content/ko/tools/lighthouse/index.md]: The Chrome product name shouldn't be translated, this goes for all instances throughout the document.

File: src/content/ko/tools/lighthouse/index.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+보고서는 계정에 시크릿 gist로 저장되므로 나중에 쉽게 보고서를 삭제하거나 업데이트할 수 있습니다.
+Github를 데이터 스토리지로 사용한다는 건 무료 버전 컨트롤을 의미합니다.
+
+## 기여하기
+
+Lighthouse는 오픈소스이며 [기여를 환영합니다](https://github.com/GoogleChrome/lighthouse/blob/master/CONTRIBUTING.md).
+저장소의 [이슈 트래커](https://github.com/GoogleChrome/lighthouse/issues)를 확인하여 수정할 수 있는 버그를 찾거나, 심사를 만들거나 개선할 수 있습니다.
+이슈 트래커는 심사 메트릭스, 새로운 심사를 위한 아이디어, Lighthouse와 관련한 기타 사항을 논의하기에 좋은 장소입니다.
+
+[예제]: https://github.com/justinribeiro/lighthouse-mocha-example/blob/master/test/lighthouse-tests.js
```
Comment (petele) [inline src/content/ko/tools/lighthouse/index.md]: This line needs to remain as is, it should not be translated, as it's used in a link further up.

**Comment by techhtml** [comment]
@petele fix contents depend on your review!

**Comment by paulirish** [comment]
@techhtml this is so great! thanks for contributing this translation!

File: src/content/ko/tools/lighthouse/index.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+.lighthouse-install-container {
+  text-align: center;
+  margin: 32px;
+}
+</style>
+
+[Lighthouse](/web/tools/lighthouse/)는 웹 앱의 품질을 개선하는
+[오픈 소스](https://github.com/GoogleChrome/lighthouse) 자동화 도구입니다.
+
+Lighthouse는 Chrome 확장프로그램, 커맨드 라인에서, 노드 모듈에서 프로그램으로도 사용할 수 있습니다.
```
Comment (cwdoh) [inline src/content/ko/tools/lighthouse/index.md]: `프로그램으로도` -> `프로그램 형태로`

File: src/content/ko/tools/lighthouse/index.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+Lighthouse에 확인할 URL을 지정하고, 페이지에 대한 테스트를 실행한 다음, 페이지에 대한 보고서를 생성합니다.
```
Comment (cwdoh) [inline src/content/ko/tools/lighthouse/index.md]: 실행한 다음 -> 실행하면

**Comment by cwdoh** [comment]
Wow, it's like lightning! I didn't know this was already merged. :)

**Pull request merged**